packages/components/src/components/textarea/textarea.stories.ts

```diff
@@ -0,0 +1,371 @@
+    `;
+  }
+};
+
+/**
+ * Use the `placeholder` attribute to add a placeholder.
+ */
+
+export const Placeholders = {
+  args: overrideArgs([{ type: 'attribute', name: 'placeholder', value: 'Type something' }]),
+  render: (args: any) => {
+    return html`
+      <div class="w-[250px]">
+        ${generateTemplate({
+          args
+        })}
+      </div>
+    `;
+  }
+};
+
+/**
+ * Use the disabled attribute to disable a textarea. All interaction is disabled and no events will be fired.
+ */
+export const Disabled = {
+  parameters: {
+    controls: {
+      exclude: ['disabled']
+    }
+  },
+  args: overrideArgs([
+    { type: 'attribute', name: 'value', value: 'value' },
+    { type: 'attribute', name: 'label', value: 'Label' },
+    { type: 'attribute', name: 'help-text', value: 'help-text' }
+  ]),
+  render: (args: any) => {
+    return html`
+      <div class="w-[250px]">
+        ${generateTemplate({
+          constants: [{ type: 'attribute', name: 'disabled', value: true }],
+          args
+        })}
+      </div>
+    `;
+  }
+};
+
+/**
+ * Use the readonly attribute to render a textarea as readonly.  Interaction is enabled, but the textarea cannot be edited.  Events will be fired.
+ */
+export const Readonly = {
+  parameters: {
+    controls: {
+      exclude: ['readonly']
+    }
+  },
+  args: overrideArgs([
+    { type: 'attribute', name: 'value', value: 'value' },
+    { type: 'attribute', name: 'label', value: 'Label' },
+    { type: 'attribute', name: 'help-text', value: 'help-text' }
+  ]),
+  render: (args: any) => {
+    return html`
+      <div class="w-[250px]">
+        ${generateTemplate({
+          constants: [{ type: 'attribute', name: 'readonly', value: true }],
+          args
+        })}
+      </div>
+    `;
+  }
+};
+
+/**
+ * This shows sd-textarea in its various sizes.
+ */
+
+export const Sizes = {
+  parameters: {
+    controls: {
+      exclude: ['size']
+    }
+  },
+  args: overrideArgs([
+    { type: 'attribute', name: 'value', value: 'value' },
+    { type: 'attribute', name: 'label', value: 'Label' },
+    { type: 'attribute', name: 'help-text', value: 'help-text' }
+  ]),
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: { type: 'attribute', name: 'size' }
+      },
+      args
+    });
+  }
+};
+
+export const Types = {
+  parameters: {
+    controls: {
+      include: ['size', 'disabled', 'readonly']
+    }
+  },
+  render: (args: any) => {
+    return html`
+      <div class="w-[370px]">
+        <div class="mb-2">
+          ${generateTemplate({
+            constants: [
+              { type: 'attribute', name: 'type', value: 'text' },
+              { type: 'attribute', name: 'label', value: 'Text' },
+              { type: 'attribute', name: 'placeholder', value: '.*' },
+              { type: 'attribute', name: 'help-text', value: 'default type' }
+            ],
+            args
+          })}
+        </div>
+      </div>
+    `;
+  }
+};
```

Comment:
Do we need this story? Seems like it's just showing what we already see in the other ones, or am I missing something 😃

Reply:
was just copied from input